src/main/java/com/klin/holoItems/utility/ReflectionUtils.java

```diff
@@ -36,4 +86,20 @@ public static int getRepairCost(ItemStack stack) {
         }
         return 0;
     }
+
+    /**
```

Comment:
`ItemStack::finishUsingItem` handles everything, including decrementing the stack and applying effects